apps/web/src/components/SidebarData.tsx

```diff
@@ -0,0 +1,38 @@
+import { SidebarButtonProps, ButtonKind } from './Interfaces';
```

Comment:
Could we remove this file and move it to src/common/constants.ts

Reply:
Moved to src/common/constants.ts, thanks Karthikk!